### ISSUE TITLE
accessibility of some important sws windows

### ISSUE THREAD

**Issue by ranetto** [jcsteh/osara#1056]
Good morning, lately I have started using regions especially to copy entire areas of the song in reaper, this allows me not to have to select tracks or items with the problem that it must always be done in track order otherwise the items and tracks do not return to the their original position in the pasting, for example if I have a piano, a bass and several drum tracks I should always re-paste starting from the piano, but if there is an empty track between them all this doesn't work, then with the playlist editor of the regions by gluing the regions themselves this does not happen. I had to create some scripts for jaws that visualize me and allow me to interact with the columns of the windows themselves, unfortunately I don't know how to write for NVDA, these windows have various problems because the columns and their coordinates are populated only when there are elements. Well, after this brief introduction to explain myself better, I wonder if it is up to me to be able to make these two or three windows accessible such as the region playlist editor, the live window, the cycle action editor window, etc. Among other things, these windows are all very similar with the same concepts of classic columns as syslistview32 except for the way they are populated, which however often contain within them a second listview which deals with the contents themselves. In short, to not make it too long, could Dare do something about these windows? Thanks in advance!

**Comment by jcsteh** [comment]
OSARA can't fix this. This needs to be fixed in SWS.
